src/server/api/routers/eventRoute.ts

```diff
@@ -3,69 +3,75 @@ import { z } from "zod";
 import { createTRPCRouter, publicProcedure, protectedProcedure } from "../trpc";
 
 import { PrismaClient } from "@prisma/client";
-import Email from "next-auth/providers/email";
+import { TRPCError, initTRPC } from '@trpc/server';
 
 const prisma = new PrismaClient();
 
+type User = {
+    id: string
+    name: string
+    email: string
+    emailVerified: Date
+    image: string
+    createdAt: Date
+    updatedAt: Date
+  }
+
+type Event = {
+    id: string
+    occuringAt: Date
+    ownerID: string
+    address: string
+    createdAt: Date
+    updatedAt: Date
+}
+
+type Participate = {
+    eventID: string
+    userID: Date
+    timeSlots: JSON
+}
+
```

Comment:
This may be trivial but perhaps it's more suitable to use `interface` instead of `type`? @RaymondWHZ what are your takes on this